docs/tutorials/05_problem_transformers.ipynb

```diff
@@ -0,0 +1,461 @@
+      "   [6.64581730e-01 2.59298923e-16]]]\n",
+      "\n",
+      "\n",
+      " [[[8.68926823e-17 6.64581730e-01]\n",
+      "   [1.80931200e-01 1.82411770e-16]]\n",
+      "\n",
+      "  [[1.80931200e-01 2.57172666e-16]\n",
+      "   [7.20426423e-17 6.98573723e-01]]]]\n"
+     ]
+    }
+   ],
+   "source": [
+    "mo_problem = basis_transformer.transform(ao_problem)\n",
+    "print(mo_problem.basis)\n",
+    "\n",
+    "mo_hamil = mo_problem.hamiltonian\n",
+    "print(mo_hamil.electronic_integrals.alpha)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c90aa467",
+   "metadata": {},
+   "source": [
+    "If you need to construct your `BasisTransformer` manually, you can do this like so:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "6cea99d6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "from qiskit_nature.second_q.operators import ElectronicIntegrals\n",
+    "from qiskit_nature.second_q.problems import ElectronicBasis\n",
+    "from qiskit_nature.second_q.transformers import BasisTransformer\n",
+    "\n",
+    "ao2mo_alpha = np.random.random((2, 2))\n",
+    "ao2mo_beta = np.random.random((2, 2))\n",
+    "\n",
+    "basis_transformer = BasisTransformer(\n",
+    "    ElectronicBasis.AO,\n",
+    "    ElectronicBasis.MO,\n",
+    "    ElectronicIntegrals.from_raw_integrals(ao2mo_alpha, h1_b=ao2mo_beta),\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "17922900",
+   "metadata": {},
+   "source": [
+    "## The `FreezeCoreTransformer`\n",
+    "\n",
+    "This transformer provides you with the very simple means to freeze the _core orbitals_ of your molecular system. It requires your problem to contain the `.molecule` attribute from which it can extract the atomic information necessary to perform this Hilbert space reduction."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "4a85694a",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Molecule:\n",
+      "\tMultiplicity: 1\n",
+      "\tCharge: 0\n",
+      "\tUnit: Bohr\n",
+      "\tGeometry:\n",
+      "\t\tLi\t(0.0, 0.0, 0.0)\n",
+      "\t\tH\t(0.0, 0.0, 2.8345891868475928)\n",
+      "\tMasses:\n",
+      "\t\tLi\t7\n",
+      "\t\tH\t1\n",
+      "(2, 2)\n",
+      "6\n"
+     ]
+    }
+   ],
+   "source": [
+    "from qiskit_nature.second_q.drivers import PySCFDriver\n",
+    "\n",
+    "driver = PySCFDriver(atom=\"Li 0 0 0; H 0 0 1.5\")\n",
+    "\n",
+    "full_problem = driver.run()\n",
+    "print(full_problem.molecule)\n",
+    "print(full_problem.num_particles)\n",
+    "print(full_problem.num_spatial_orbitals)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "9a37235a",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "(1, 1)\n",
+      "5\n"
```

Comment:
Could we note that for LiH the core orbitals is just a the single lowest energy orbital. So as a result the output is one less orbital and the two electrons from that orbital are removed as well. Can we say something about the frozen core energy we extract which gets added back to the overall result at the end.